lib/resque.rb

```diff
@@ -152,6 +152,19 @@ def redis_id
   # Set or retrieve the current logger object
   attr_accessor :logger
 
+  DEFAULT_HEARTBEAT_INTERVAL = 60
+  DEFAULT_PRUNE_INTERVAL = DEFAULT_HEARTBEAT_INTERVAL * 5
+
+  attr_writer :heartbeat_interval
+  def heartbeat_interval
+    @heartbeat_interval || DEFAULT_HEARTBEAT_INTERVAL
+  end
+
+  attr_writer :prune_interval
```

Comment:
Ask yourself, you wrote this :-) But the reader has a fallback to the default, so I guess that's why?